src/collections/_documentation/performance-monitoring/performance/event-detail.md

```diff
@@ -22,18 +22,24 @@ Currently, only root transactions are searchable. Any span data that inherits fr
 
 [{% asset performance/span-details.png alt="Span detail view shows the span id, trace id, parent span id, and other data such as tags." %}]({% asset performance/span-details.png @path %})
 
+You can view span data using either of these methods:
+
+1. Click the "Related Transaction" button from the Issue Details page to open the Transaction summary. This will maintain the context of the current error event.
```

Comment:
@MimiDumpling this button has now been moved to the "trace" context panel as "View Summary": 
<img width="1329" alt="Screen Shot 2020-06-26 at 5 53 53 PM" src="https://user-images.githubusercontent.com/139499/85904295-511d9900-b7d6-11ea-8fc1-faf6908a76db.png">